.github/workflows/label_conflicts.yml

```
@@ -0,0 +1,20 @@
+name: Auto Label Conflicts
+on:
+  push:
+    branches: [master]
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [master]
+
+jobs:
+  auto-label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: prince-chrismc/label-merge-conflicts-action@v1
+        with:
+          conflict_label_name: "has conflicts"
```

A label needs to be selected/created

alternatively?

```suggestion
          conflict_label_name: "needs rebase"
```

---

I'm against "needs rebase" - this will make people think they need to rebase, but after rebase it's impossible to review the PR in incremental mode.
`has conflicts` or `merge conflicts` work fine for me.

---

That's a good point! what about `merge master`?

---

Also works for me quite well.